xml/obs_ag_overview_filesystem.xml

```diff
@@ -647,13 +647,58 @@ auto_cleanup_after_days: 30
          <para>see Reference Guide for more information</para>
         </entry>
        </row>
+       <row>
+        <entry align="left" valign="top">
```

Comment:
What is this section for?

Reply:
The table that shows the features flags?